src/components/HelpSection.vue

```diff
@@ -383,6 +383,50 @@ export default {
             isModal5Active: false,
             isModal6Active: false
         }
+    },
+    watch: {
```

Comment:
Please try to find a better way to do this, instead of having six separate functions, I would rather have one function and pass the `eventLabel` as a parameter.